src/OrchardCore.Modules/OrchardCore.Users/ViewModels/ForgotPasswordViewModel.cs

```diff
@@ -1,10 +1,27 @@
+using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Localization;
+using MimeKit;
 
 namespace OrchardCore.Users.ViewModels
 {
-    public class ForgotPasswordViewModel
+    public class ForgotPasswordViewModel : IValidatableObject
     {
         [Required]
-        public string UserIdentifier { get; set; }
+        public string Email { get; set; }
+
+        public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
+        {
+            var S = validationContext.GetService<IStringLocalizer<ForgotPasswordViewModel>>();
+            if (string.IsNullOrWhiteSpace(Email))
+            {
+                yield return new ValidationResult(S["Email is required."], new[] { nameof(Email) });
+            }
+            else if (Email.IndexOf('@') == -1 || !MailboxAddress.TryParse(Email, out var emailAddress))
```

Comment:
We already have `IEmailAddressValidator` on Infrastructure APIs that could be used here